docs/solution-architecture/systemContext.md

```diff
@@ -38,6 +39,7 @@ C4Context
     Rel(Employee, ZAC, "Handles cases")
 
     Rel(OpenFormulieren, OpenNotificaties, "Uses", "ZGW Notificaties API")
+    Rel(OpenArchiefbeheer, OpenNotificaties, "Uses", "ZGW Notificaties API")
```

Comment:
Is this true? Does OAB connect with OpenNotificaties? I was under the impression that OAB only integrates with OpenZaak (and OpenZaak sends notifications).

Reply:
Good question, will fix this in a next update if needed.